packages/google_maps_flutter/google_maps_flutter_ios/ios/google_maps_flutter_ios.podspec

```diff
@@ -25,8 +25,15 @@ Downloaded by pub (not CocoaPods).
   # Versions earlier than 8.4 can't be supported because that's the first version
   # that supports privacy manifests.
   s.dependency 'GoogleMaps', '>= 8.4', '< 10.0'
+  s.dependency 'Google-Maps-iOS-Utils', '>= 5.0', '< 7.0'
```

Comment:
@stuartmorgan You had a comment about this, but it's way buried now. Does this seem like a good range? v5 and v6 were released at the same time. The only difference is v6 supports GoogleMaps v9 which requires iOS 15.

See https://github.com/googlemaps/google-maps-ios-utils/issues/473

Reply:
Yes, this matches what I expected based on the comment there.

Reply:
But this line should get a comment explaining that 5.x is for GoogleMaps 8.x and 6.x is for GoogleMaps 9.x, so it's clear how in the file itself how this line relates to the line above and its explanatory comment.